ground/gcs/src/plugins/boards_dtf/dtfc.h

```diff
@@ -36,37 +36,35 @@
 
 #include <coreplugin/iboardtype.h>
 
-class IBoardType;
-
 class Dtfc : public Core::IBoardType
 {
 public:
     Dtfc();
     virtual ~Dtfc();
 
-    virtual QString shortName();
-    virtual QString boardDescription();
-    virtual bool queryCapabilities(BoardCapabilities capability);
-    virtual QPixmap getBoardPicture();
-    virtual QString getHwUAVO();
-    virtual int queryMaxGyroRate();
+    QString shortName();
+    QString boardDescription();
```

Comment:
I figured the Seppuku implementation was the way to go, as that was the newest one. Will revert!